docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -18,6 +18,11 @@ Moreover, they accept as first parameter the name of the database towards which
 
 Through this implementation, we can use these procedures in a cluster environment,
 by leveraging the cluster routing mechanism.
+
+Moreover, to use these procedures the user must have the `admin` role, 
+otherwise the procedure throws with an error `permission has not been granted for user 'xxx'`,
+to prevent anyone from adding a trigger that could escalate privileges. 
```

Comment:
We discussed that we usually don't explain the background of the privileges in the documentation, so I would remove this section.